app/models/bulkrax/csv_entry.rb

```diff
@@ -18,15 +18,14 @@ def self.read_data(path)
                encoding: 'utf-8')
     end
 
-    def self.data_for_entry(data, path = nil)
+    def self.data_for_entry(data)
       # If a multi-line CSV data is passed, grab the first row
       data = data.first if data.is_a?(CSV::Table)
       raw_data = data.to_h
       # If the collection field mapping is not 'collection', add 'collection' - the parser needs it
       raw_data[:collection] = raw_data[collection_field.to_sym] if raw_data.keys.include?(collection_field.to_sym) && collection_field != 'collection'
       # If the children field mapping is not 'children', add 'children' - the parser needs it
       raw_data[:children] = raw_data[collection_field.to_sym] if raw_data.keys.include?(children_field.to_sym) && children_field != 'children'
-      raw_data[:file] = record_file_paths(path) if raw_data[:file].blank?
```

Comment:
when bagit was used, this was causing files to be added twice (due to new bagit based adding in bagit parser).  when bagit was not used, this is where the weird "[]" were coming from.